File: lecture_001/largestofthree.cpp
#include<iostream>
//verified
using namespace std;
int main(int args, char** argv)
{
    int a, b, c;
    cout<<"enter a:"<<endl;
    cin>>a;
    cout<<"enter b:"<<endl;
    cin>>b;
    cout<<"enter c:"<<endl;
    cin>>c;

    if (a>b && b>c){
        cout<<"largest is a:"<<a<<endl;
    }
    else if (b>a && b>c){
        cout<<"largest is b:"<<b<<endl;
    }
    else{
        cout<<"largest is c:"<<c<<endl;
    }
    return 0;
}

File: lecture_001/largestofthree.java
import java.util.Scanner;
//verified
public class largestofthree{
    public static void main(String[] args){
        Scanner scn=new Scanner(System.in);
        int a, b, c;

        System.out.print("enter a: ");
        a = scn.nextInt();
        System.out.print("enter b: ");
        b = scn.nextInt();
        System.out.print("enter c: ");
        c = scn.nextInt();
        if (a>b && b>c){
            System.out.print("largest is a:" + a);
        }
        else if (b>a && b>c){
            System.out.print("largest is b:" + b);
        }
        else{
            System.out.print("largest is c:" + c);
        }
    }
}